bool start_routing
int8 tb_id
geometry_msgs/Point goal_point
---
geometry_msgs/Point distance
